src/Compilers/Core/MSBuildTask/Shared/Microsoft.CSharp.Core.targets

```diff
@@ -95,7 +95,7 @@
           FileAlignment="$(FileAlignment)"
           GenerateFullPaths="$(GenerateFullPaths)"
           HighEntropyVA="$(HighEntropyVA)"
-          Instrument="$(Instrument)"
+          InstrumentationKinds="$(Instrument)"
```

Comment:
After lots of back and forth, the current deliberate decision is that Instrument is a property. There is precedent for multiple items being combined in a property, e.g. NoWarn.

Also, "InstrumentKinds" above will revert to "Instrument" to match the spelling of the command-line option.